packages/design-system/src/components/datalist/datalist.js

```diff
@@ -198,7 +199,7 @@ const Datalist = forwardRef(function Datalist(
   const selectedOption = primaryOptions.find(({ id }) => id === selectedId);
   // In case of isInline, the list is displayed with 'absolute' positioning instead of using a separate popup.
   return (
-    <Container>
+    <Container className={className}>
```

Comment:
This is for making styled-components work, as it requires the `className` prop to be passed to the component when doing `styled(DataList.DropDown)`